### ISSUE TITLE
Open source Open-WA

### ISSUE THREAD

**Issue by Hentai-sebpai** [open-wa/wa-automate-nodejs#1416]
Going to lock this self-rightous convo until I can fully reply.

TLDR: No

_Originally posted by @smashah in https://github.com/open-wa/wa-automate-nodejs/issues/1414#issuecomment-787057629_

I seem to feel like this issue was not resolved. 

tried to make a bot using open-wa but after every idea I come up with the first thought is will it need a license key 
I mean if you have to pay for most of the features why would this even be on Github? The post story and message non contact I understand but for features like reply with reply with sticker and fake image thumbnail only for license key holders is preposterous :v

If you aren't gonna make this fully open source just rename this project to Closed-WA and take it off of Github LMAO

**Comment by smashah** [comment]
I was going to reply to that issue once the real issues were resolved by later tonight. If anyone else makes another issue regarding this then they will be blocked.

> Closed-WA

Has a good ring to it. Might actually change the name one day. Anybody not new to this project knows I consider people's circumstances if they ask for a key or if they contribute a good idea to the project. Unfortunately, moaning and whining preclude anyone from any such goodwill.
